### ISSUE TITLE
Cutoff materials not working properly

### ISSUE THREAD

**Issue by hybridherbst** [0beqz/realism-effects#2]
When clicking through the samples, I noted the Swordsman looks wrong around the cape / cutout parts:

![20230304-213731_chrome](https://user-images.githubusercontent.com/2693840/222927635-67ca6486-3b95-480c-9a7e-d5a3a9e1a0f5.gif)

I think either transparency or cutout may not be working correctly with SSGI on.

**Comment by 0beqz** [comment]
True, the reason why it's not supported yet is because transparency didn't have a priority and as I wasn't sure which method would be the best to deal with transparency without losing performance due to extra renders.

Right now I'm experimenting with alpha hashing in the G-buffers which allows for transparency but results in flickering at the moment due to how the denoiser works. I'm looking for a performant way to get rid of the flickering and fully support transparency.

**Comment by 0beqz** [comment]
This is still an on-going issue as, with how the pipeline is designed, transparency is quite performance-draining because it would require a second G-Buffer pass as it seems.
So I'll have to look into the best solution for this pipeline. Right now, the most performant solution could be rendering all transparent materials either invisible or opaque and then accumulate them as the camera stopped moving (due to detecting disocclusions and reprojecting in the TR pass being also complicated with transparent materials).

**Comment by hybridherbst** [comment]
I think the above sample could pass with just cutout (instead of opacity blending), but agreed in the general case it's hard...
